backend/src/api/model/user.rs

```diff
@@ -17,7 +17,7 @@ impl User {
         &self.username
     }
 
-    fn email(&self) -> &str {
+    fn email(&self) -> &Option<String> {
         &self.email
     }
```

Comment:
It's just a tiny thing, but this would be slightly preferred:

```suggestion
    fn email(&self) -> Option<&str> {
        self.email.as_deref()
    }
```